sal_interfaces/MTDome/MTDome_Commands.xml

```diff
@@ -80,42 +86,42 @@
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_stopLouvers</EFDB_Topic>
-    <Description>Stop all louvers.</Description>
+    <Description>Stop all louvers. This should also be used for an emergency stop.</Description>
```

Comment:
I find this additional text confusing -- both on this command and all the other device-specific stop commands below. What sort of emergency requires stopping only the louvers (or other single device)?

My first thought would be to simply remove the text. If you expand the description of the "stop" and "goStationary" commands then that should suffice.

Two other options include changing the description to something like "Quickly stop all louvers" or to state that this is one of the things the "stop" command does. Both imply "there is no quicker way to stop the louvers", which I suspect is the point.